draft-ietf-dnssd-privreq.xml

```diff
@@ -817,7 +824,128 @@ This draft does not require any IANA action.
   </front>
 </reference>
 
+<reference anchor="Wu16" target="https://arxiv.org/pdf/1604.06959.pdf%22">
+  <front>
+    <title>Privacy, discovery, and authentication for the internet of things</title>
+    <author initials='D.' surname='Wu' fullname='David Wu'>
+      <organization>Stanford University</organization>
+    </author>
+    <author initials='A.' surname='Taly' fullname='Ankur Taly'>
+      <organization>Google</organization>
+    </author>
+    <author initials='A.' surname='Shankar' fullname='Asim Shankar'>
+      <organization>Google</organization>
+    </author>
+    <author initials='D.' surname='Boneh' fullname='Dan Boneh'>
+      <organization>Stanford University</organization>
+    </author>
+    <date year="2016"/>
+  </front>
+</reference>
+
+<reference anchor="SIGMA" target="http://link.springer.com/content/pdf/10.1007/978-3-540-45146-4_24.pdf">
+  <front>
+    <title>SIGMA: The 'SIGn-and-MAc'approach to authenticated Diffie-Hellman and its use in the IKE protocols</title>
+    <author initials='H.' surname='Krawczyk' fullname='Hugo Krawczyk'>
+      <organization>EE Department, Technion, Haifa, Israel, and IBM T.J. Watson Research Center</organization>
+    </author>
+    <date year="2003"/>
+  </front>
+</reference>
+
 </references>  
 
+  <section title="Survey of Implementations">
+
+    <t>This section surveys several private service discovery designs in the context of the threat model
+      detailed above.</t>
+
+    <section title="DNS-SD Privacy Extensions">
+      <t>Huitema and Kaiser <xref target="I-D.huitema-dnssd-privacy"></xref> decompose 
+      private service discovery into two stages: (1) identify specific peers offering 
```

Comment:
The reference should be to I-D.ietf-dnssd-privacy. "huitema-dnssd-privacy" is an older version of that draft, before adoption as a work item by the DNSSD WG. It might be worth noting that this draft went through last call in the WG, and that the objections were raised after this last call completed. As far as IETF process is concerned, this is the solution adopted so far by the WG.

Reply:
Will fix.

Reply:
Done.